client/state/sites/test/reducer.js

```diff
@@ -94,14 +103,14 @@ describe( 'reducer', () => {
 					}
 				} );
 			} );
-			it.skip( 'when state has validation errors on DESERIALIZE it returns initial state', () => {
+			it( 'when state has validation errors on DESERIALIZE it returns initial state', () => {
```

Comment:
Just a minor vocabulary remark, conventionally `it` is provided an argument that causes the line to flow as human-readable text, i.e.:

`it( 'returns initial state when state has validation errors on DESERIALIZE', () => {`